frontend/src/pages/profile/components/user-activity/styles.module.scss

```diff
@@ -1,9 +1,13 @@
 .wrapper {
+  display: block;
   grid-area: activity;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 40px;
```

Comment:
This alignment doesn't make anything because display: block doesn't have such properties